features/createroom/impl/src/main/kotlin/io/element/android/features/createroom/impl/CreateRoomFlowNode.kt

```diff
@@ -58,23 +60,34 @@ class CreateRoomFlowNode @AssistedInject constructor(
 
         @Parcelize
         object NewRoom : NavTarget
+
+        @Parcelize
+        data class ConfigureRoom(val users: List<MatrixUser>) : NavTarget
     }
 
     override fun resolve(navTarget: NavTarget, buildContext: BuildContext): Node {
         return when (navTarget) {
             NavTarget.Root -> {
-                val callback = object : CreateRoomRootNode.Callback {
+                createNode<CreateRoomRootNode>(context = buildContext, plugins = listOf(object : CreateRoomRootNode.Callback {
```

Comment:
Can you just split the creation of the callback?

Reply:
Done https://github.com/vector-im/element-x-android/pull/308/commits/803dafca9a825644849bf14787d6c75bcc503e06